game-engine/core/src/main/java/za/co/entelect/challenge/core/entities/GameDetails.java

```diff
@@ -43,9 +43,9 @@ public GameDetails(int round) {
                 .filter(bt -> !buildingTypesWithoutStats.contains(bt))
                 .forEach(bt -> buildingsStats.put(bt, BuildingFactory.createBuildingStats(bt)));
 
-        ironCurtainStats.put("price", 150);
-        ironCurtainStats.put("activeRounds", 6);
-        ironCurtainStats.put("resetPeriod", 50);
-        ironCurtainStats.put("constructionScore", 20);
+        ironCurtainStats.put("price", GameConfig.getIroncurtainPrice());
+        ironCurtainStats.put("activeRounds", GameConfig.getIroncurtainActiveRounds());
+        ironCurtainStats.put("resetPeriod", GameConfig.getIroncurtainResetPeriod());
+        ironCurtainStats.put("constructionScore", GameConfig.getIroncurtainConstructionScore());
```

Comment:
Have we updated the GameConfig file with these added setting in a previous PR?

Reply:
Yes. the gameconfig had them before, but I missed this part as I tested the json capability, forgot to finalize this part